src/ethereum/harvest.finance/contracts.ts

```diff
@@ -7,13 +7,10 @@ const contracts = {
         abi: HarvestController,
         address: "0x3cC47874dC50D98425ec79e647d83495637C55e3",
     },
-    HarvestDeposit: {
-        abi: HarvestDeposit,
-    },
-    HarvestFarm: {
-        abi: HarvestFarm,
-    },
-    "weth": {
+    HarvestDeposit: { abi: HarvestDeposit },
+    HarvestFarm: {abi: HarvestFarm },
```

Comment:
Please get all the pools from https://github.com/Opty-Fi/team-defi-adapters/blob/master/helpers/data/defiPools.json#L280 and https://github.com/Opty-Fi/team-defi-adapters/blob/master/helpers/data/defiPools.json#L480

Reply:
Done